backend/test/search.test.ts

```diff
@@ -25,9 +67,20 @@ describe('search', () => {
   });
   describe('search', () => {
     it('search by global admin should work', async () => {
-      searchDomains.mockImplementation(() => ({
-        body: [1, 2, 3]
-      }));
+      searchDomains.mockImplementation(() => {
```

Comment:
It might be clearer, instead of using numTimesSearched, to just chain `mockImplementationOnce` as shown here: https://jestjs.io/docs/en/mock-function-api#mockfnmockimplementationoncefn